src/main/java/net/sigmalab/maven/plugin/jira/VersionComparator.java

```diff
@@ -6,13 +6,13 @@
 
 public class VersionComparator implements Comparator<Version> {
 
```

Comment:
**Replace all tab characters in this file by sequences of white-spaces.**

Developers should not need to configure the tab width of their text editors in order to be able to read source code.

So the use of tabulation character must be banned.